sdk/script/management-sdk.ts

```diff
@@ -1,30 +1,23 @@
 import * as base64 from "base-64";
 import crypto = require("crypto");
+import * as fs from "fs";
 import * as os from "os";
+import * as path from "path";
 import Q = require("q");
+import slash = require("slash");
 import superagent = require("superagent");
+import * as recursiveFs from "recursive-fs";
+import * as yazl from "yazl";
 
 import Promise = Q.Promise;
 
-import { AccessKey, AccessKeyRequest, Account, App, CodePushError, CollaboratorMap, CollaboratorProperties, Deployment, DeploymentMetrics, Headers, Package, PackageInfo, ServerAccessKey, Session, UpdateMetrics } from "./types";
+import { AccessKey, AccessKeyRequest, Account, App, CodePushError, CollaboratorMap, CollaboratorProperties, Deployment, DeploymentMetrics, Headers, IPackageFile, Package, PackageInfo, ServerAccessKey, Session, UpdateMetrics } from "./types";
 
 var superproxy = require("superagent-proxy");
 superproxy(superagent);
 
 var packageJson = require("../package.json");
 
-declare var fs: any;
-
-if (typeof window === "undefined") {
-    fs = require("fs");
-} else {
-    fs = {
-        createReadStream: (fileOrPath: string): void => {
-            throw new Error("Tried to call a node fs function from the browser.");
-        }
-    }
-}
```

Comment:
Okay, it's possible this has gotten out of date. In that case I'm fine with this just being removed, and then we can patch it up later for the browser when/if it's ever needed!